### PR TITLE
Update README with new config file name.

### PR DESCRIPTION
**Pull request by jezdez**

### CODE REVIEW

**Comment by codingjoe** [comment]
Hi @jezdez I am not so sure about this. I actually do like `.checks.yml` as a default. But I am willing to change my mind on that. Anyhow, I would like to keep the instructions super instructive. So I don't know if providing options (even though the exist) is a good idea here. In a longer documentation I see that as helpful but maybe not here.

What are your thoughts on this?

**Comment by jezdez** [comment]
Sure thing, to reiterate what I mentioned in the threads for the Jazzband integration: given the sheer number of dotfiles floating around in the root directory of repos, and the rise of GitHub apps that support them, I think having explicit naming for the file lowers the barrier of maintenance and improves the self-documenting character of a file named about the service it relies on. I understand that naming is hard, so treat my opinion with a grain of salt -- as always. Thanks for working on FussyFox in the first place!

File: README.md [hunk 1]
```diff
@@ -16,9 +16,9 @@ The setup is simple. Just add a file config file to your repository specifying
 which linters you want to run.
 
 The configuration is stored in a simple YAML format in a file named
-`.checks.yml`.
+`.fussyfox.yml` (or the old but deprecated name `.checks.yml`).
```
Comment (codingjoe) [inline README.md]:
```suggestion
`.fussyfox.yml`.
```

**Review by codingjoe — CHANGES_REQUESTED** [review]
I see your point. Let's change the default, but let's keep it simple and make only one suggestion on how to do it.

**Review by codingjoe — APPROVED** [review]

**Pull request merged**